doc/CMakeOptions.rst

```diff
@@ -0,0 +1,43 @@
+============================================  =======================================
+Pennylane Lightning ``Cmake`` option           Description
+============================================  =======================================
+``-DENABLE_BLAS=OFF``                         Enable BLAS
+``-DENABLE_CLANG_TIDY=OFF``                   Enable clang-tidy build checks
+``-DENABLE_COVERAGE=OFF``                     Enable code coverage
+``-DENABLE_GATE_DISPATCHER=ON``               Enable gate kernel dispatching on AVX/AVX2/AVX512
+``-DENABLE_LAPACK=OFF``                       Enable compilation with scipy/LAPACK
+``-DENABLE_NATIVE=OFF``                       Enable native CPU build tuning
+``-DENABLE_OPENMP=ON``                        Enable OpenMP
+``-DENABLE_PYTHON=ON``                        Enable compilation of the Python module
+``-DENABLE_WARNINGS=ON``                      Enable warnings
+``-DLQ_ENABLE_KERNEL_AVX_STREAMING=OFF``      Enable AVX2/512 streaming operations for gate kernels
+``-DLQ_ENABLE_KERNEL_OMP=OFF``                Enable OpenMP pragmas for gate kernels
+``-DPL_DISABLE_CUDA_SAFETY=OFF``              Build without CUDA call safety checks
```

Comment:
Does that belong to Lightning, or L-GPU? I feel like it should be L-GPU/L-Tensor.

Reply:
Yes, you are right, I will remove it :smile: